### PR TITLE
feat(docs):[TRI-1654] Add section in building block view for subproto…

### PR DESCRIPTION
**Pull request by ds-jhartmann**
…colBody usage

### CODE REVIEW

File: docs/src/docs/arc42/building-block-view/level-2.adoc [hunk 1]
```diff
@@ -85,7 +85,7 @@ include::../../../uml-diagrams/building-block-view/level-2-int-transfer-process-
 |The DigitalTwinRegistryFacade calls the DigitalTwinRegistry to retrieve data form the AAS registry and transforms the response to internal data models.
 
 |SubmodelFacade
-|The SubmodelFacade calls the EDC to retrieve data from the submodel server and transforms the response to internal data models.
+|The SubmodelFacade handles EDC contract negotiation, calls the EDC dataplane to retrieve data from the submodel server and transforms the response to internal data models.
```
Comment (mkanal) [inline docs/src/docs/arc42/building-block-view/level-2.adoc:88]:
```suggestion
|The SubmodelFacade handles EDC contract negotiations and is responsible to request the EDC dataplane to retrieve data from the submodel server and transforms the response to the internal data models. <jobResponse>
```

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc [hunk 2]
```diff
@@ -74,6 +74,14 @@ The IRS gives its users the ability to manage, create and delete complex policie
 
 The structure of a Policy that can be stored in storage can be easily viewed by using Policy Store endpoints in the published API documentation. Each policy may contain more than one permission, which in turn consists of constraints linked together by AND or OR relationships. This model provides full flexibility and control over stored access and use policies.
 
+== Digital Twin / EDC requirements
+
+In order to work with the decentral network approach, IRS requires the Digital Twin to contain a "subprotocolBody" in each of the submodelDescriptor endpoints. This "subprotocolBody" has to contain the "id" of the EDC asset, as well as the "dspEndpoint" of the EDC, separated by a semicolon (e.g. `"subprotocolBody": "id=123;dspEndpoint=http://edc.control.plane/api/v1/dsp"`).
+
+The "dspEndpoint" is used to request the EDC catalog of the dataprovider and the "id" to filter for the exact asset inside this catalog.
```
Comment (mkanal) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:81]:
```suggestion
The "**dspEndpoint**" is used to request the EDC catalog of the dataprovider and the "id" to filter for the exact asset inside this catalog.
```

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc [hunk 3]
```diff
@@ -74,6 +74,14 @@ The IRS gives its users the ability to manage, create and delete complex policie
...
+
+If the "dspEndpoint" is not present, every available EDC endpoint in DiscoveryService will be queried until a asset with the "id" can be found.
```
Comment (mkanal) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:83]: ****
```suggestion
If the "**dspEndpoint**" is not present, every available EDC endpoint in DiscoveryService will be queried until a asset with the "id" can be found.
```

**Review by mkanal — APPROVED** [review]

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc (hunk 3 above)
Comment (ds-jhartmann) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:83]:
```suggestion
If the `"dspEndpoint"` is not present, every available EDC endpoint in DiscoveryService will be queried until a asset with the `"id"` can be found.
```

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc (hunk 2 above)
Comment (ds-jhartmann) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:81]:
```suggestion
The `"dspEndpoint"` is used to request the EDC catalog of the dataprovider and the `"id"` to filter for the exact asset inside this catalog.
```

File: docs/src/docs/arc42/cross-cutting/under-the-hood.adoc [hunk 4]
```diff
@@ -74,6 +74,14 @@ The IRS gives its users the ability to manage, create and delete complex policie
 
 The structure of a Policy that can be stored in storage can be easily viewed by using Policy Store endpoints in the published API documentation. Each policy may contain more than one permission, which in turn consists of constraints linked together by AND or OR relationships. This model provides full flexibility and control over stored access and use policies.
 
+== Digital Twin / EDC requirements
+
+In order to work with the decentral network approach, IRS requires the Digital Twin to contain a "subprotocolBody" in each of the submodelDescriptor endpoints. This "subprotocolBody" has to contain the "id" of the EDC asset, as well as the "dspEndpoint" of the EDC, separated by a semicolon (e.g. `"subprotocolBody": "id=123;dspEndpoint=http://edc.control.plane/api/v1/dsp"`).
```
Comment (ds-jhartmann) [inline docs/src/docs/arc42/cross-cutting/under-the-hood.adoc:79]:
```suggestion
In order to work with the decentral network approach, IRS requires the Digital Twin to contain a `"subprotocolBody"` in each of the submodelDescriptor endpoints. This `"subprotocolBody"` has to contain the `"id"` of the EDC asset, as well as the `"dspEndpoint"` of the EDC, separated by a semicolon (e.g. `"subprotocolBody": "id=123;dspEndpoint=http://edc.control.plane/api/v1/dsp"`).
```

File: docs/src/docs/arc42/building-block-view/level-2.adoc (hunk 1 above)
Comment (ds-jhartmann) [inline docs/src/docs/arc42/building-block-view/level-2.adoc:88]:
```suggestion
|The SubmodelFacade handles EDC contract negotiations and is responsible for the EDC dataplane requests to retrieve data from the submodel servers.
```

**Review by mkanal — DISMISSED** [review]

**Review by mkanal — APPROVED** [review]

**Pull request merged**